revup/revup.py

```diff
@@ -217,9 +217,10 @@ async def main() -> int:
         "cherry-pick",
         add_help=False,
     )
+    commit_parser = subparsers.add_parser("commit", add_help=False)
```

Comment:
rather than add new parser, use aliases=["commit"]

Reply:
although you might have to dig up the original invocation as so https://stackoverflow.com/questions/59493715/resolve-argparse-alias-back-to-the-original-command but I think this could be a useful addition to the RevupArgParser class

Reply:
Unless I'm misunderstanding something, I don't think that's necessary for this use case since we explicitly want to distinguish between the original "amend" and the alias "commit" to switch behavior

Reply:
hmm i was assuming args.cmd would always end up as "amend" but looks like that's not the case. that's convenient then